apps/platform/trpc/routers/orgRouter/users/invitesRouter.ts

```diff
@@ -227,17 +245,25 @@ export const invitesRouter = router({
         const newInvitePublicId = typeIdGenerator('orgInvitations');
         const newInviteToken = nanoIdToken();
 
-        await db.insert(orgInvitations).values({
-          publicId: newInvitePublicId,
-          orgId: orgId,
-          invitedByOrgMemberId: orgMemberId,
-          orgMemberId: +newOrgMemberResponse.insertId,
-          role: newOrgMember.role,
-          email: notification?.notificationEmailAddress ?? null,
-          inviteToken: newInviteToken,
-          invitedOrgMemberProfileId: orgMemberProfileId,
-          expiresAt: new Date(Date.now() + 7 * 24 * 60 * 60 * 1000) // 7 Days from now
-        });
+        await db
+          .insert(orgInvitations)
+          .values({
+            publicId: newInvitePublicId,
+            orgId: orgId,
+            invitedByOrgMemberId: orgMemberId,
+            orgMemberId: +newOrgMemberResponse.insertId,
+            role: newOrgMember.role,
+            email: notification?.notificationEmailAddress ?? null,
+            inviteToken: newInviteToken,
+            invitedOrgMemberProfileId: orgMemberProfileId,
+            expiresAt: new Date(Date.now() + 7 * 24 * 60 * 60 * 1000) // 7 Days from now
+          })
+          .catch((err) => {
```

Comment:
You don't need catch here, trpc handles that part if you are gonna throw error anyways. It's actually better to throw the Original error as it helps in debugging

Reply:
fixed.